src/librustc_trans/debuginfo/mod.rs

```diff
@@ -168,7 +168,7 @@ pub fn finalize(cx: &CrateContext) {
     }
 
     debug!("finalize");
-    let _ = compile_unit_metadata(cx);
+    //let _ = compile_unit_metadata(cx);
```

Comment:
It will definitely need to change, as we changed the function.
However, the new version might even work with old LLVM, doesn't it?